### DISCUSSION TITLE
Handling WhatsApp images

### DISCUSSION THREAD

**Discussion by xplosionmind** [photoprism/photoprism#2355] [Q&A]
Images and videos received via WhatsApp are stripped of all their metadata, including their original date. While on Android the received images filename [at least contain the received date](https://github.com/photoprism/photoprism/issues/1102), for images received on an iOS device it is absolutely impossible to have any sort of date or time metadata.

As a result, while importing on PhotoPrism images I received via WhatsApp I find my gallery flooded with a ton of images in “Review” and I literally have no clue how to at least sort them (I can live with them not containing any date, but not with them appearing all together in an unordered chunk).

How can I sort this out?  
How do you handle WhatsApp images?

Thank you very, very much!

**Comment by graciousgrey** [comment]
I personally delete most of them as they have low quality. For sharing photos with friends I use PhotoPrism or a messenger that allows to send files without removing metadata. That's why my review section mostly consists of files I don't want to keep.
If I want to keep some of them I approve them, add them to albums and/or change the date manually.

If in general you want to keep all of those you can disable the review feature in settings. This way all photos from review will appear in normal search results without approval.

If you set up [automatic syncing from your phone](https://docs.photoprism.app/user-guide/sync/mobile-devices/) new messenger-photos will be imported on a daily basis and get the import date (the date they have been sent to you) set as date. This way they are kind of sorted when filtering for newest/oldest.

**Reply by xplosionmind** [reply]
Thank you very much for sharing. Unfortunately, it is quite hard to make everyone share pictures with better, technologically more advanced, systems like PhotoPrism. I still receive plenty of relevant images from WhatsApp and chat software…

In any case, I will try and look for the optimal way to handle my current 1500+ pictures in review.

**Reply by graciousgrey** [reply]
> Unfortunately, it is quite hard to make everyone share pictures with better, technologically more advanced, systems like PhotoPrism

#557 

Might be easier to convince people once uploads are possible in shared albums. But yes there will always be people preferring WhatsApp.

> In any case, I will try and look for the optimal way to handle my current 1500+ pictures in review

If you come up with ideas that would make this easier, let us know :)
